### ISSUE TITLE
Should it have a name itself and can submit like a normal iron-input?

### ISSUE THREAD

**Issue by hankphung** [ellipticaljs/paper-autocomplete#43]
Hi Guys,
I'm about to use your component as a replace for the `combobox` because of many limitation. But I have to add an `iron-input` following with it to make the value present in form data on submit. So it will better if this have `name` attribute and can be submit also.
Thank a lot.

**Comment by jhuesos** [comment]
Hi,

It is interesting that we all missed such an important attribute: name. 
I will be adding it, hopefully, today. 

In the meantime, there is another option that is not being displayed currently in the page with demos and documentation ( I need to do that as well... :( ) You can now use the autocomplete functionality with any paper-input or input field of your choice. This component has been splitted in two: paper-autocomplete and paper-autocomplete-suggestions.

If you want to see how it works, do the following:
```
1. Clone this repo
2. npm install && bower install
3. polymer serve -o 
4. Navigate to paper-autocomplete-suggestions documentation to see an example.
```

Let me know if that works for you.

**Comment by hankphung** [comment]
I have forked your repository and try working around. I did seen the `paper-autocomplete-suggestions`, so interested in. 
Thank you.

**Comment by jhuesos** [comment]
Hi,

Are you sure you have the latest version? You should be able to see the following file in your fork:

https://github.com/ellipticaljs/paper-autocomplete/blob/master/paper-autocomplete-suggestions.html

**Comment by hankphung** [comment]
Yes, I see it.

**Comment by jhuesos** [comment]
I have been extremely busy this week, but I will try to add the name option to `paper-autocomplete` this week. Let me know how it goes.

**Comment by hankphung** [comment]
I will open a pull request.
#45

**Comment by tachyon1337** [comment]
>It is interesting that we all missed such an important attribute: name.

Because it was originally written when polymer didn't support paper-input form submissions.

**Comment by jhuesos** [comment]
Fixed in version v2.2.0
